plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/ComponentComponent.groovy

```diff
@@ -66,17 +64,17 @@ extends DirectComponentSupport
     }
     StorageTx storageTx = repository.facet(StorageFacet).openTx()
     try {
-      OrientVertex component = storageTx.findComponent(new ORecordId(componentId), storageTx.getBucket())
+      ComponentImpl component = storageTx.findComponent(new ORecordId(componentId), storageTx.getBucket())
```

Comment:
D'oh, good catch, will fix. The impls were intentionally package-scoped, but this is groovy code so it escaped my notice. IDEA's "extract interface" is awesome, but does seem to prefer to keep old declarations going at the impl vs the new interface as one might expect.